src/controllers/user.ts

```diff
@@ -0,0 +1,23 @@
+import UserModel from "../models/user.model";
+import { Request, Response, NextFunction } from "express";
+
+export const getOtherProfile = (req: Request, res: Response, next: NextFunction) => {
+  UserModel.findOne({ username: req.params.username })
```

Comment:
Validate the input before you query please :D There is a lot of examples in the validator folder.

Reply:
![image](https://user-images.githubusercontent.com/38537908/117572411-131a2280-b0d3-11eb-907d-8fa8d4fb88d7.png) 
hope this works for me, i always forget to add :D validation logic is added, sorry about that